Zend/tests/short_function/short_function_declaration.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+Short function declaration
+--FILE--
+<?php
+function foo(): void = echo 123;
+var_dump(foo()); // NULL
+
+function bar(): int = return 456;
+echo bar() . PHP_EOL;
+
+// function buz(): int = 456; // TODO: must be the same as return 456;
+// echo buz() . PHP_EOL;
+
+function compare(int $what, int $with): string = switch(true) {
```

Comment:
I'm actually kind of shocked that a switch works here, since I thought it was all statements.  I would expect `match()` to be much more commonly used here.

Checking `switch()` is fine, but please include a `match()` test as well, as that seems the more common use case.

Reply:
It doesn't really work. If this were a match, the value would fail to be returned, and we'd just return `null`.